tools/build_defs/framework.bzl

```diff
@@ -52,8 +52,8 @@ CC_EXTERNAL_RULE_ATTRIBUTES = {
     #
     # Optional part of the shell script to be added after the make commands
     "postfix_script": attr.string(mandatory = False),
-    # Optinal make commands, defaults to ["make", "make install"]
-    "make_commands": attr.string_list(mandatory = False, default = ["make", "make install"]),
+    # Optinal make commands, defaults to ["make -j`nproc`", "make install"]
+    "make_commands": attr.string_list(mandatory = False, default = ["make -j`nproc`", "make install"]),
```

Comment:
I'm not sure building in parallel is a good default. Many packages have broken makefiles when built in parallel, also bazel assumes that each task running is using a single core and by default when building using local execution strategy there are no groups applied and so this causes the make command to over provision itself on resources.

I'd suggest that if you want this behaviour in your own use that you define a macro that sets make_commands to this value?

Reply:
@jsharpe I see, take a look at #447 and I will create another pr to fix the `increment_pkg_config_path`?

Reply:
#447 is next on my list of PRs to look at; however I don't currently have any uses of `cmake_external` or `configure_make` that make use of dependencies via pkg-config - do you have a suggestion of a project that does to use to test this PR against?

Reply:
#447 is now merged. Could you create a PR for the `increment_pkg_config_path` fix?

Reply:
@whs-dot-hk did you want to put in a PR for the fixes to `increment_pkg_config_path`?